### PR TITLE
DCMTK: Add version 3.6.7

### LINKED ISSUE #5283: [Request]: DCMTK
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Non-GUI tool
- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 500 stars and/or 150 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

DCMTK 3.6.7

### Description

DCMTK is a collection of libraries and applications implementing large parts the DICOM standard. These tools are a set of command-line utilities for working with DICOM (Digital Imaging and Communications in Medicine, see also https://en.wikipedia.org/wiki/DICOM).

### Homepage

https://dcmtk.org/en/dcmtk/

### Download Link(s)

https://dicom.offis.de/download/dcmtk/dcmtk367/bin/dcmtk-3.6.7-win64-dynamic.zip

### Some Indication of Popularity/Repute

These tools have been around ~20+ years and are fairly prevalent within the community.

Official GitHub repository mirror exists at: https://github.com/DCMTK/dcmtk, which currently has 635 stars and 280 forks.

DCMTK is currently available on Chocolatey and Homebrew in their main repositories
- https://community.chocolatey.org/packages/dcmtk (currently indicates ~3.1k installs of the current version)
- https://formulae.brew.sh/formula/dcmtk (currently indicates ~1.2k installs in the past year.

### PR DESCRIPTION
**Pull request by neandrake**
Adds DCMTK 3.6.7 for 32bit and 64bit.

Closes #5283

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/dcmtk.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "version": "3.6.7",
+    "description": "DCMTK is a collection of libraries and applications implementing large parts the DICOM standard.",
+    "homepage": "https://dcmtk.org/en/dcmtk/",
+    "license": "BSD-3-Clause",
+    "env_add_path": "./bin/",
```
Comment (tech189) [inline bucket/dcmtk.json:6]: Looks good to me, could you just move `env_add_path` to after `architecture` but before `checkver`? (As per the order here https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets)

**Review by tech189 — CHANGES_REQUESTED** [review]

File: bucket/dcmtk.json (hunk 1 above)
Comment (neandrake) [reply bucket/dcmtk.json]: Certainly I will update the ordering of the fields. I did initially look to follow the ordering from that documentation however I would note that `architecture` does not currently appear in the preferred ordering listed on that wiki though it is apparent on the linked template. Could the wiki be updated to make it more explicit where `architecture` should belong?

Comment (tech189) [reply bucket/dcmtk.json]: Yes that's a good point, I'll make a change to that help page.

**Review by tech189 — APPROVED** [review]

File: bucket/dcmtk.json (hunk 1 above)
Comment (neandrake) [reply bucket/dcmtk.json]: Thank you @tech189!

**Pull request merged**